### DISCUSSION TITLE
Possibility for configuring custom InconsistentLogPropertyNaming

### DISCUSSION THREAD

**Discussion by corey-re-leased** [olsh/resharper-structured-logging#50] [Ideas]
Hi,

Within our team, we use prefixes to our Log Properties to allow for filtering in Serilog Seq.

This is demonstrated below:
`_logger.LogInformation("Starting Execution of {RLS_FunctionName}", functionName);`

Unfortunately, this is leading to InconsistentLogPropertyNaming message: suggested name is **RlsFunctionName**

I was wondering whether you could implement some sort of configuration to exclude some custom LogPropertyNaming patterns?

Thanks,
Corey

**Comment by olsh** [comment]
Hi Corey,

So, I suppose it should be an array of patterns (regexes) that if a message matches one of these regexes then the analyzer should be disabled for the message template?

**Reply by corey-re-leased** [reply]
Hi Oleg,

Thanks for getting back to me.

I've discussed this with the team and they agree on the array of patterns (regexes) approach - we appreciate your time here :)

Kind Regards,
Corey

**Comment by olsh** [comment]
@corey-re-leased 
I've added the feature in this release
https://github.com/olsh/resharper-structured-logging/releases/tag/2022.1.2.261
Thank you! 👍🏻
